reach

```diff
@@ -115,10 +115,147 @@ run_s () {
   sh -ac "$TMP/out.sh" "$0"
 }
 
+gen_reachc_id () {
+  REACH_ENV_DIR="${HOME}/.config/reach"
+  REACH_ENV_FILE="${REACH_ENV_DIR}/env"
+  MULT=$(shuf -i 1-100 -n 1)
+  MAX=776627963145224
+  PRE_REACHC_ID=$(echo $(($(shuf -i 1-$MAX -n 1 )*$MULT))| md5sum | head -c 30 | tr [a-z] [A-Z])
+
+  mkdir -p $REACH_ENV_DIR
+
+  if [ ! -e $REACH_ENV_FILE ]; then
+    echo "REACHC_ID=${PRE_REACHC_ID}" > $REACH_ENV_FILE
+  elif ( ! grep REACHC_ID $REACH_ENV_FILE > /dev/null 2>&1 ); then
+    echo "REACHC_ID=${PRE_REACHC_ID}" >> $REACH_ENV_FILE
+  fi
+}
+
+call_home () {
+
+  START_TIME=$1
+  ELAPSED_TIME=$2
```

Comment:
Please go back through Jira/Slack/GitHub/etc and ensure what you're tracking is up-to-date.

https://github.com/reach-sh/lambdas/blob/master/lambdas/core-analytics/src/index.js

Reply:
This might have been an old column on dynamo. I'll delete that. And change the template.